pkg/kapp/cmd/app/deploy_flags.go

```diff
@@ -34,7 +34,7 @@ func (s *DeployFlags) Set(cmd *cobra.Command) {
 	cmd.Flags().StringVar(&s.IntoNamespace, "into-ns", "", "Place resources into namespace")
 	cmd.Flags().StringSliceVar(&s.MapNamespaces, "map-ns", nil, "Map resources from one namespace into another (could be specified multiple times)")
 
-	cmd.Flags().BoolVarP(&s.Patch, "patch", "p", false, "Add or update provided resources")
+	cmd.Flags().BoolVarP(&s.Patch, "patch", "p", false, "Add or update existing resources only, never delete any")
```

Comment:
I wonder if just "Add or update existing resources only" is sufficient, but this definitely makes the flags purpose a lot clearer.